torch/profiler/pattern_matcher.py

```diff
@@ -1,103 +0,0 @@
-from torch.autograd.profiler import profile
```

Comment:
Curious that this shows up...

Reply:
I drop an accidental commit, seems to be related.